benchopt/dataset_util.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from scipy import sparse
+from bz2 import BZ2Decompressor
+from os.path import join as pjoin
```

Comment:
I prefer `import os` and using `os.path.join`.
It is more verbose but avoid custom names or having to import each command separately.

Reply:
addressed